working/macros/feature-specification.md

```diff
@@ -118,6 +118,28 @@ Most of the time, like here, a macro takes the arguments you pass it and
 interpolates them back into code that it generates, so passing the arguments as
 code is what you want.
 
+### Identifier arguments
+
+If you want to be able to introspect on an identifier passed in to you, you can
+do that as well, consider the following:
+
+```dart
+@GenerateSerializers(MyType)
+library my.library;
+
+// Generated
+class MyTypeSerializer implements Serializer<MyType> {
+  // ....
+}
+
+class MyTypeDeserializer implements Deserializer<MyType> {
+  // ....
+}
+```
+
+Here the macro takes an `Identifier` argument, and introspects on it to know
+how to generate the desired serialization and deserialization classes.
```

Comment:
Might be good to show a little more than `// ....` to hint at how the introspection is involved. Right now, it just looks like `MyType` is propagated over to the type argument in the generated class, which doesn't require introspection.